### ISSUE TITLE
WebDAV File Transfer and Folder Creation Forbidden Using Client

### ISSUE THREAD

**Issue by Gadget-Guru** [redmatrix/hubzilla#454]
Dev branch file transfer _to server_ and folder creation is forbidden when using webdav clients nemo, nautilus and CyberDuck. Uploaded the same file via browser and all is fine.

I'm reproducing the same issue (HTTP Client Error: 403 Forbidden) at a second dev hub.

Here's the log each time I attempt to create a new folder:
```
 [0] => Cron
)

2016-07-18 22:20:58:LOG_INFO::Cron.php:24:run: cron: Already running
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory /
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: /
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory /dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: /dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:733:FileData: /dav/seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 2
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:472:getDir: paths: Array
(
[0] => seth
)

2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory /
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: /
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_DEBUG::Master.php:26:Release: Master: release: Array
(
[0] => Cron
)

2016-07-18 22:21:01:LOG_INFO::Cron.php:24:run: cron: Already running
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory /dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: /dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:733:FileData: /dav/seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 2
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:472:getDir: paths: Array
(
[0] => seth
)

2016-07-18 22:21:01:LOG_DEBUG::Master.php:26:Release: Master: release: Array
(
[0] => Cron
)

2016-07-18 22:21:01:LOG_INFO::Cron.php:24:run: cron: Already running
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory /
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: /
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory /dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: /dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:733:FileData: /dav/seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 2
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:472:getDir: paths: Array
(
[0] => seth
)

2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: Untitled Folder
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:733:FileData: seth/Untitled Folder
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:821:FileData: not found seth/Untitled Folder
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:362:createDirectory: create directory Untitled Folder
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory /
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: /
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory /dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: /dav
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:733:FileData: /dav/seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:53:__construct: directory seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:451:getDir: GetDir: seth
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 2
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:472:getDir: paths: Array
(
[0] => seth
)

2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:107:getChild: Untitled Folder
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:733:FileData: seth/Untitled Folder
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:Directory.php:821:FileData: not found seth/Untitled Folder
2016-07-18 22:21:01:LOG_DEBUG::Master.php:26:Release: Master: release: Array
(
[0] => Cron
)

2016-07-18 22:21:01:LOG_INFO::Cron.php:24:run: cron: Already running
2016-07-18 22:21:01:LOG_DEBUG::Master.php:26:Release: Master: release: Array
(
[0] => Cron
)

2016-07-18 22:21:01:LOG_INFO::Cron.php:24:run: cron: Already running
```

**Comment by redmatrix** [comment]
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:271:log: channel_name
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:272:log: channel_id 0
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:273:log: channel_hash
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:274:log: observer
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:275:log: owner_id 2
2016-07-18 22:21:01:LOG_INFO:jce0cld26c35gnie0m4uh6qea0:BasicAuth.php:276:log: owner_nick

The issue here is that you are not logged in. Are there any earlier auth log messages which could help explain why not?

**Comment by git-marijus** [comment]
@Gadget-Guru have you by chance `Can view my file storage and photos` set to anything less than `Anybody on the internet`?

**Comment by Gadget-Guru** [comment]
It is set to Anybody on the internet

**Comment by git-marijus** [comment]
fixed in 77a021025ff4c8dac6fb416031257e952de89914
